### PR TITLE
Better Debugs

### PR DESCRIPTION
**Pull request by adrian-enspired**
fixes https://nexcess.atlassian.net/browse/NSD-12290

**DEPENDS ON** https://github.com/nexcess/nexcess-php-sdk/pull/32
~DO NOT MERGE: needs to `composer update nexcess/nexcess-php-sdk` once nexcess-php-sdk/32 is merged~

example command output with `-vvv`:
https://gist.github.com/adrian-enspired/a8d5035d370e8d582f9d6e57a862aa8d

### CODE REVIEW

File: src/Console.php [hunk 1]
```diff
@@ -45,6 +45,15 @@
  */
 class Console extends SymfonyApplication {
 
+  /** @var int Verbose output. */
```
Comment (calevans) [inline src/Console.php]: Why are we redefining constants here? The symfony ones should be used.

Comment (adrian-enspired) [reply src/Console.php]: use-specific, easier to remember, easier to type

**Comment by adrian-enspired** [comment]
updated sdk; ready to merge

File: src/Console.php (hunk 1 above)
Comment (calevans) [reply src/Console.php]: unnecessary.

**Review by calevans — APPROVED** [review]
even tho i disagree. :)

**Pull request merged**